clang/test/Preprocessor/predefined-macros-hlsl.hlsl

```diff
@@ -12,7 +12,7 @@
 // HALF: #define __HLSL_ENABLE_16_BIT 1
 // NOHALF-NOT: __HLSL_ENABLE_16_BIT
 
-// CHECK: #define __HLSL_VERSION 2021
+// CHECK: #define __HLSL_VERSION 2028
```

Comment:
I do not understand where this 2028 is coming from. Is 202x intended to be 2028?

Reply:
202x doesn't have a year yet, so to make it big and far out I gave it 2028 and 202y has 2029... these will change before we actually ship.